subscription/management/commands/notify_changed_erps.py

```diff
@@ -54,7 +54,11 @@ def get_notifications(self, hours, now=None):
             erp = versioning.extract_online_erp(version)
             if not erp:
                 continue
-            subscribers = [sub.user for sub in ErpSubscription.objects.subscribers(erp)]
+            subscribers = [
+                sub.user
+                for sub in ErpSubscription.objects.subscribers(erp)
+                if sub.user.is_active
```

Comment:
We should rather enforce the `is_active` filter in the `ErpSubscription.objects.subscribers` queryset method.